### PR TITLE
prepare NCP opeartor 3.2.0.1 release

### PR DESCRIPTION
**Pull request by timdengyun**
In preparation of NCP opeartor 3.2.0.1 release:
Update year to 2021 for LICENSE and Notice
Update VERSION to 3.2.0.1

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 nsx-container-plugin-operator
 
-Copyright (c) 2020 VMware, Inc.  All rights reserved.				
+Copyright (c) 2021 VMware, Inc.  All rights reserved.
```
Comment (salv-orlando) [inline LICENSE:3]: I'm not sure if this needs to be updated for each release, usually we don't. But I don't think it'll do any harm updating it.

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-3.2.0-dev
+3.2.0.1
```
Comment (salv-orlando) [inline VERSION:1]: thanks!

**Review by salv-orlando — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (timdengyun) [reply LICENSE:3]: Yes, I'm also not very sure on this update.  I did it as my previous experience.  Thanks for comments.

File: VERSION (hunk 2 above)
Comment (timdengyun) [reply VERSION:1]: Welcome. It's aligned with NCP release version.

**Pull request merged**